src/utils/constants.js

```diff
@@ -18,3 +18,8 @@ export const AccountTypes = {
   TREZOR: 'trezor',
   METAMASK: 'browser'
 };
+
+export const Routes = {
+  MCD: 'borrow',
+  DSR: 'save'
+};
```

Comment:
Not sure if this is the best place for these and that the names for the keys are ok?

Reply:
The key names might as well be `BORROW` and `SAVE` too